### PR TITLE
Updated Courses.

### PR DESCRIPTION
**Pull request by ashwini0529**

### CODE REVIEW

File: Python/courses.md [hunk 1]
```diff
@@ -7,4 +7,5 @@
 - [Python with Django](https://goo.gl/DVM0m5) - Django over the python.
 - [Programming fundamental with python](https://goo.gl/JEdLMw) - Intermediate level course on python data structure and basic topics.
 - [Developing Scalable app with Python](https://goo.gl/EwWZDW) - How to use Google app engine along with python to create scalable webApp. 
-
+-[Python Ultimate Course- Udemy](https://goo.gl/GRuvrL) - Python course by Udemy to build concepts.
+-[Networking in Python](http://www.tutorialspoint.com/python/python_networking.htm) - A good tutorial to implement networking in python.
```
Comment (PareshMayani) [inline Python/courses.md]: I think this is not a course but useful tutorials. Please correct me if I am wrong! If this is tutorial link then please include it under "useful_links" section!

Comment (ashwini0529) [inline Python/courses.md]: As discussed with @PSanni, he suggested me to add the Python Udemy course under course section. Networking in Python is a tutorial, but could be included as a course.

Comment (PareshMayani) [inline Python/courses.md]: I am talking about [this link](http://www.tutorialspoint.com/python/python_networking.htm) which is actually a tutorial, not a complete course!

Comment (ashwini0529) [inline Python/courses.md]: Okay. I'll add it under Useful links.

Comment (PareshMayani) [inline Python/courses.md]: Thanks!

**Pull request closed without merging**